src/main/java/practice/StreamPractice.java

```diff
@@ -14,7 +18,12 @@
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        return numbers.stream()
+                .map(stringNumber -> stringNumber.split(","))
+                .flatMap(stringNumber -> Arrays.stream(stringNumber))
```

Comment:
can we only use flatMap?
```suggestion
                .flatMap(stringNumber -> Arrays.stream(stringNumber.split(",")))
```